### PR TITLE
x11docker: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

This is a complex command with a lot of concepts of Linux and security, so I left here a few examples and the README has a detailed explanation about this command.

### CODE REVIEW

File: pages/common/x11docker.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# x11docker
+
+> Run GUI applications and desktops in docker giving focus on security.
+> See also `xephyr`.
+> More information: <https://github.com/mviereck/x11docker>.
+
+- Launch VLC in a container:
+
+`x11docker --pulseaudio --share={{$HOME/Videos}} {{jess/vlc}}`
```
Comment (marchersimon) [inline pages/common/x11docker.md:9]: What does `jess` mean?

File: pages/common/x11docker.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Launch Xfce in a window:
+
+`x11docker --desktop {{x11docker/lxde}}`
```
Comment (marchersimon) [inline pages/common/x11docker.md:13]: Wouldn't that start `Lxde` in a window?

File: pages/common/x11docker.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/x11docker.md:9]: That's the container image name. https://hub.docker.com/r/jess/vlc.  @jessfraz is famous for her Docker work ;)

File: pages/common/x11docker.md (hunk 2 above)
Comment (bl-ue) [reply pages/common/x11docker.md:13]: That must be I typo.
```suggestion
- Launch LDXE in a window:

`x11docker --desktop {{x11docker/lxde}}`
```

File: pages/common/x11docker.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/x11docker.md:9]: As someone, who dosn't know much about docker, the examples descriptions are a bit too generic / the examples are a bit too specific. But again, if you know docker you probably also understand all the examples, so I think it's fine.

Comment (bl-ue) [reply pages/common/x11docker.md:9]: Well, this example is about lauching VLC 🤔

File: pages/common/x11docker.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/x11docker.md:13]: A last minute change lol

**Review by marchersimon — APPROVED** [review]

File: pages/common/x11docker.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# x11docker
+
+> Run GUI applications and desktops in docker giving focus on security.
```
Comment (bl-ue) [inline pages/common/x11docker.md:3]: The text taken verbatim from the repo doesn't sound very good.  How about this?
```suggestion
> Securely run GUI applications and desktop UIs in Docker containers.
```

**Review by Waples — APPROVED** [review]

**Comment by navarroaxel** [comment]
Hey @Waples, we remove the auto generated commit message body from GitHub when we squash and merge.

**Comment by Waples** [comment]
I just pressed the squash and merge button on my tablet,didn't add any comments

**Comment by navarroaxel** [comment]
@Waples I think there is an option in the GitHub Android app
Currently, the git log for this commit `71c0279659c1ec1d7e161cb4751fe091058917ae` is:
```
x11docker: add page (#5835)
    
* x11docker: add page
    
* Update pages/common/x11docker.md

* Update pages/common/x11docker.md
    
Co-authored-by: bl-ue <54780737+bl-ue@users.noreply.github.com>
  
Co-authored-by: bl-ue <54780737+bl-ue@users.noreply.github.com>
```
instead it should be just the commit subject without a message body:
```
```
x11docker: add page (#5835)
```

**Comment by Waples** [comment]
Yep I just noticed that it has a default

**Comment by navarroaxel** [comment]
it's just noise in the git log, it doesn't add a useful information. I really the idea of a GitHub setting to disable this auto generated text.

**Comment by bl-ue** [comment]
Yes, that would be nice.  With GitLab, you can configure the default commit message when applying suggestions, but that's not quite the same here 🤔

The extended message body, while not great, isn't much of a problem, so don't worry that much guys 😉

**Pull request merged**